wp-content/plugins/core/src/Templates/Controllers/Content/Panels/CardGrid.php

```diff
@@ -82,18 +82,23 @@ protected function get_cardgrid_attributes() {
 
 	protected function get_card_image( $image_id ) {
 		if ( empty( $image_id ) ) {
-			return false;
+			return '';
+		}
+
+		try {
```

Comment:
I like this pattern; do we think there's any value in also logging the Exception for troubleshooting purposes? Otherwise it just sorta fails silently.